ray-operator/controllers/ray/rayjob_controller_test.go

```diff
@@ -112,580 +112,665 @@ func rayJobTemplate(name string, namespace string) *rayv1.RayJob {
 	}
 }
 
-var _ = Context("RayJob in K8sJobMode", func() {
-	Describe("RayJob SubmitterConfig BackoffLimit", Ordered, func() {
-		ctx := context.Background()
-		namespace := "default"
-		rayJobWithDefaultSubmitterConfigBackoffLimit := rayJobTemplate("rayjob-default", namespace)
-		rayJobWithNonDefaultSubmitterConfigBackoffLimit := rayJobTemplate("rayjob-non-default", namespace)
-		rayJobWithNonDefaultSubmitterConfigBackoffLimit.Spec.SubmitterConfig = &rayv1.SubmitterConfig{
-			BackoffLimit: ptr.To[int32](88),
-		}
-		rayJobs := make(map[*rayv1.RayJob]int32)
-		rayJobs[rayJobWithDefaultSubmitterConfigBackoffLimit] = int32(2)
-		rayJobs[rayJobWithNonDefaultSubmitterConfigBackoffLimit] = int32(88)
-
-		It("Verify RayJob spec", func() {
-			for rayJob := range rayJobs {
-				// Make sure the submission mode is K8sJobMode.
-				Expect(rayJob.Spec.SubmissionMode).To(Equal(rayv1.K8sJobMode))
+var _ = Context("RayJob with different submission modes", func() {
+	Context("RayJob in K8sJobMode", func() {
```

Comment:
For reviewers: Please note that all the code in `Context("RayJob in K8sJobMode")` is simply indented. It may appear as though there are many changes, but in reality, nothing has actually changed.